assemble/src/main/scripts/create-jshell.sh

```diff
@@ -36,6 +36,31 @@ function addAccumuloAPI() {
   echo
 }
 
+function addClientBuild() {
+cat <<EOF
+  URL clientPropUrl = 
+    AccumuloClient.class.getClassLoader().getResource("accumulo-client.properties");
+  AccumuloClient client = null;
+  String name; String zk; String principal; String token;
+      
+  // Does Accumulo properties exists?
+  if (clientPropUrl != null) {
+    var prop = new Properties(); 
+
+    // Load in Accumulo properties
+    try (var in = clientPropUrl.openStream()) {
+      prop.load(in);
+    }
+    // Build Accumulo Client after try-with-resources is closed
+    System.out.println("Building Accumulo client using '" + clientPropUrl + "'\n");
```

Comment:
In case it fails to load, it would be nice to see the URL printed so the user can troubleshoot. So, that should be printed first.

```suggestion
    // Load in Accumulo properties
    System.out.println("Building Accumulo client using '" + clientPropUrl + "'\n");
    try (var in = clientPropUrl.openStream()) {
      prop.load(in);
    }
    // Build Accumulo Client after try-with-resources is closed
```